packages/bierzo-wallet/src/routes/payment/components/ReceiverAddress/index.tsx

```diff
@@ -18,8 +18,8 @@ interface Props {
 }
 
 const ReceiverAddress = ({ form, selectedChainCodec }: Props): JSX.Element => {
-  const recipientValidator: FieldValidator = React.useMemo(() => {
-    const validator = (value: string): string | undefined => {
+  const recipientValidator: FieldValidator<string | undefined> = React.useMemo(() => {
```

Comment:
You can use `InputValue` here as well, if I understand you correctly.

Reply:
Right, thanks, will update